README.md

```diff
@@ -463,21 +484,36 @@ This filter provides the capability to enable a feature for a target audience. A
                             "RolloutPercentage": 50
                         }
                     ],
-                    "DefaultRolloutPercentage": 20
+                    "DefaultRolloutPercentage": 20,
+                    "Exclusion": {
+                        "Users": [
+                            "Ross"
+                        ],
+                        "Groups": [
+                            "Ring2"
+                        ]
+                    }
                 }
             }
         }
     ]
 }
 ```
 
+### RequirementType

```

Comment:
This section is talking mostly about feature filters. I think RequirementType can be moved to the top, closer to Feature Flag Declaration section.